concent_api/concent_api/settings/base.py

```diff
@@ -238,3 +238,6 @@
 # - "Gatekeeper":  An internal helper that validates file transfer tokens.
 # - "admin-panel": Django admin panel that provides access to database content and service statistics.
 # CONCENT_FEATURES = []
+
+# URL format: 'protocol://<user>:<password>@<hostname>:<port>/<virtual host>'
+#CELERY_BROKER_URL = ''
```

Comment:
Isn't pylint complaining about missing space between # and commented code ?

Reply:
Fixed